.github/workflows/examples.yaml

```diff
@@ -84,6 +84,7 @@ jobs:
 
       - name: Build TinyGo examples
         run: make build.examples.tinygo
+        if: matrix.go-version != '1.20' # fails with TinyGo v0.32.0
```

Comment:
Idk how we feel about skipping building tests with Go 1.20 + TinyGo v0.32.0, as that seems to be an unsupported combination.